_posts/2020-07-05-raw-image-streaming.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Raw image streaming available in Metal3"
+date: 2020-07-05
+draft: false
+categories: ["metal3", "baremetal", "raw image", "image streaming"]
+author: Maël Kimmerlin
+---
+
+Metal3 supports multiple types of images for deployment, the most
+popular being QCOW2. We have recently added support for a feature of Ironic
```

Comment:
should we add `QCOW2` in yaspeller.json as well?

Reply:
the spellcheck did not fail on it, so I guess it is not required